src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -97,27 +105,71 @@ describe('handle-key-press', () => {
   });
 
   describe('handleKeyPress', () => {
-    const rowIndex = 0;
-    const colIndex = 0;
-    const evt = {
-      key: 'ArrowDown',
-      stopPropagation: sinon.spy(),
-      preventDefault: sinon.spy(),
-    };
-    evt.target = {
-      blur: sinon.spy(),
-      setAttribute: sinon.spy(),
-    };
-    const rootElement = {
-      getElementsByClassName: () => [{ getElementsByClassName: () => [{ focus: () => {}, setAttribute: () => {} }] }],
-    };
-
-    it('should prevent default behavior and remove current focus', () => {
-      handleKeyPress(evt, rootElement, rowIndex, colIndex);
-      expect(evt.preventDefault).to.have.been.calledOnce;
-      expect(evt.stopPropagation).to.have.been.calledOnce;
-      expect(evt.target.blur).to.have.been.calledOnce;
-      expect(evt.target.setAttribute).to.have.been.calledOnce;
+    let rowIndex;
+    let colIndex;
+    let evt = {};
+    let rootElement = {};
+    let selState = {};
+    let cell = [];
+
+    beforeEach(() => {
+      rowIndex = 0;
+      colIndex = 0;
+      evt = {
+        key: 'ArrowDown',
+        stopPropagation: sinon.spy(),
+        preventDefault: sinon.spy(),
+        target: {
+          blur: sinon.spy(),
+          setAttribute: sinon.spy(),
+          offsetParent: {
+            classList: {
+              remove: () => {},
+              contains: () => {},
+            },
+          },
+        },
+      };
+      rootElement = {
+        getElementsByClassName: () => [{ getElementsByClassName: () => [{ focus: () => {}, setAttribute: () => {} }] }],
+      };
+      selState = {
+        api: {
+          confirm: sinon.spy(),
+          cancel: sinon.spy(),
+        },
+      };
+      cell = [];
+    });
+
+    describe('ArrowDown', () => {
+      it('should prevent default behavior and remove current focus', () => {
```

Comment:
...and set it to the next cell. Test is solid, just the description is a bit misleading